integration-test/473-landuse-tier.py

```diff
@@ -13,8 +13,7 @@
 # area 30089300
 assert_has_feature(
     8, 56, 115, 'landuse',
-    { 'kind': 'national_park', 'id': -921675, 'tier': 1,
-      'min_zoom': 8 })
+    { 'kind': 'national_park', 'tier': 1, 'min_zoom': 8 })
 assert_no_matching_feature(
     7, 28, 57, 'landuse',
     { 'kind': 'national_park', 'id': -921675 })
```

Comment:
Should we also take the ID off this one? It seems to be trying to test that the particular national park starts to be shown at `z8` on the basis of its area, so it's possible that there's another, bigger one which would cause it to fail without the ID. But if we're not including the ID at `z7`...?